src/components/manage/Blocks/Search/components/SortOn.jsx

```diff
@@ -73,6 +73,7 @@ const SortOn = (props) => {
           placeholder={intl.formatMessage(messages.sortOn)}
           styles={sortOnSelectStyles}
           theme={selectTheme}
+          label={intl.formatMessage('Sort by selected option')}
```

Comment:
This is not the way to support internationalization. The message needs to be translatable to other languages. See https://6.docs.plone.org/volto/recipes/i18n.html.